docs/aks-installation.md

```diff
@@ -0,0 +1,102 @@
+
+## Creating the cluster
+
+You can use any method to create a AKS cluster. The example given here is using the Azure Cloud CLI.
+
+1. Create a AKS Cluster
+
+    ```bash
+    export RESOURCE_GROUP_NAME=aks-antrea-cluster
+    export CLUSTER_NAME=aks-antrea-cluster
+    export LOCATION=westus
+
+    az group create --name $RESOURCE_GROUP_NAME --location $LOCATION
+    az aks create \
+        --resource-group $RESOURCE_GROUP_NAME \
+        --name $CLUSTER_NAME \
+        --node-count 2 \
+        --network-plugin azure
+    ```
+
+    **Note** Do not specify network-policy option.
+
+2. Get AKS Cluster Credentials
+
+   ```bash
+   az aks get-credentials --name $CLUSTER_NAME --resource-group $RESOURCE_GROUP_NAME
+   ```
+
+3. Access your Cluster
+
+    ```bash
+    kubectl get nodes
+    NAME                                STATUS   ROLES   AGE     VERSION
+    aks-nodepool1-84330359-vmss000000   Ready    agent   6m21s   v1.16.10
+    aks-nodepool1-84330359-vmss000001   Ready    agent   6m25s   v1.16.10
+    ```
+
+
+## Deploying Antrea
+
+1. Prepare the Cluster Nodes
+
+    Deploy ``antrea-node-init`` DaemonSet to enable ``azure cni`` to operate in transparent mode.
+
+    ```bash
+    kubectl apply -f https://raw.githubusercontent.com/vmware-tanzu/antrea/master/build/yamls/antrea-aks-node-init.yml
+    ```
+
+2. Download and Update Antrea YAML
+
+    Deploy a released version of Antrea from the [list of releases](https://github.com/vmware-tanzu/antrea/releases).
+Note that AKS support was added in release 0.9.0, which means you cannot pick a release older than 0.9.0.
+For any given release `<TAG>` (e.g. `v0.9.0`), get the Antrea AKS deployment yaml at:
+
+    ````
+    https://github.com/vmware-tanzu/antrea/releases/download/<TAG>/antrea-aks.yml
+    ````
+
+    To deploy the latest version of Antrea (built from the master branch) to AKS, get the Antrea AKS deployment yaml at:
+
+    ````
+    https://raw.githubusercontent.com/vmware-tanzu/antrea/master/build/yamls/antrea-aks.yml
+    ````
+
+3. Deploy Antrea
+
+    Deploy Antrea using `kubectl apply -f antrea-aks.yml`. It will deploy a single replica of Antrea controller to the AKS cluster
+and deploy Antrea agent to every Node. After a successful deployment you should be able to see these Pods running in your cluster:
+
+    ```bash
+    $ kubectl get pods --namespace kube-system  -l app=antrea
+    NAME                                 READY   STATUS    RESTARTS   AGE
+    antrea-agent-bpj72                   2/2     Running   0          40s
+    antrea-agent-j2sjz                   2/2     Running   0          40s
+    antrea-controller-6f7468cbff-5sk4t   1/1     Running   0          43s
+    antrea-node-init-6twqg               1/1     Running   0          2m
+    antrea-node-init-mqsqr               1/1     Running   0          2m
+    ```
+
+4. Restart remaining Pods
```

Comment:
I think you use capital case for the first letter of a word, so: Remaining

Reply:
I am not using that convention. Will keep lower case

Reply:
Your other steps capitalize the first letter of all words:
1. Prepare the Cluster Nodes
3. Download and Update Antrea YAML